http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/DataprocInterpreter.scala

```diff
@@ -105,6 +105,9 @@ class DataprocInterpreter[F[_]: Parallel](
     with LazyLogging {
 
   import dbRef._
+  val isMemberofGroupDonecheckable = new DoneCheckable[Boolean] {
```

Comment:
super nit, but `isMemberofGroupDoneCheckable`? I am still getting used to scala naming conventions so feel free to ignore

Reply:
?

Reply:
`c` to `C` in `checkable`, but again I am not very familiar with naming conventions so feel free to ignore